main/main.ino

```diff
@@ -701,15 +704,24 @@ void setOTA() {
   ArduinoOTA.setPassword(ota_password);
 
   ArduinoOTA.onStart([]() {
-    Log.trace(F("Start OTA" CR));
+    Log.trace(F("Start OTA, lock other functions" CR));
+    ProcessLock = true;
```

Comment:
create a `stopProcessing` in ZgatewayBT and call it here. Same with onEnd and onError create a `startProcessing`